### PR TITLE
Combine `Reference sequence` and `HGVS` fields from Variant file to create the hgvs field in json submission object 

### LINKED ISSUE #82: Submission status is "error"...
Oopsie with the submission from yesterday:
Screenshot 2023-02-01 at 08 03 37

Any chance of seeing that error, or is it lost in a http response somewhere?

This could of course be a Scout error as well, or both, since it flagged as submitted ok in there, but let's start somewhere. 😊

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #82 

**How to prepare for test**:
- launch it locally

### How to test:
- convert the 2 demo files in a submission objecct

### Expected outcome:
- [x] Submission object should contain a complecte hgvs (refseq:hgvs)

### Review:
- [x] Code approved by
- [x] Tests executed by CR
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested locally: 🆗 


image

File: preClinVar/file_parser.py [hunk 1]
```diff
@@ -228,9 +228,10 @@ def set_item_variant_set(item, variant_dict):
     genes = variant_dict.get("Gene symbol")
     if genes:
         variant["gene"] = [{"symbol": symbol} for symbol in genes.split(";")]
+    refseq = variant_dict.get("Reference sequence")
     hgvs = variant_dict.get("HGVS")
-    if hgvs:  # A Variant should have wither HGVS (hgvs in schema)
-        variant["hgvs"] = hgvs
+    if hgvs and refseq:  # A Variant should have either HGVS (hgvs in schema)
+        variant["hgvs"] = ":".join([refseq, hgvs])
```
Comment (dnil) [inline preClinVar/file_parser.py:234]: 💯

**Review by dnil — APPROVED** [review]
Looks good to me!

**Comment by northwestwitch** [comment]
Thanks @dnil I'll release and deploy the new version both on stage and prod. Remains to understand how to amen an existing submission from Scout now! 🤔

**Pull request merged**